.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: go
+
+go:
+  - 1.x
+  - 1.7
```

Comment:
copy-paste from janus =) will wait for other comments and fix this together with other issues